### PR TITLE
DTSRD-3711

### PR DESCRIPTION
**Pull request by lukasz-wolski**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/DTSRD-3711

### Change description ###

Org chart changes (changelog 151)

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by SabinaHMCTS — APPROVED** [review]
a  COMMIT; at the end would be good

File: src/main/resources/db/migration/V1_28__org_chart_updates.sql [hunk 1]
```diff
@@ -0,0 +1,634 @@
...
+
+insert into locrefdata.service (service_id, org_unit_id, business_area_id, sub_business_area_id, jurisdiction_id, service_code,
+  service_description, service_short_description, last_update) values (
+  47,
+  (select org_unit_id from locrefdata.org_unit where description='HMCTS'),
+  (select business_area_id from locrefdata.org_business_area where description ='Civil, Family and Tribunals'),
+  (select sub_business_area_id from locrefdata.org_sub_business_area   where description ='Tribunals'),
+  (select jurisdiction_id from locrefdata.jurisdiction where description ='General Regulatory Chamber'),
+  'BAB5',
+  'Electronic Comms etc Communications, Postal Services & Network Information Systems',
```
Comment (mikebrownccd) [inline src/main/resources/db/migration/V1_28__org_chart_updates.sql:43]: I think this should be 'Electronic Communications, Postal Services & Network Information Systems'

Comment (lukasz-wolski) [reply src/main/resources/db/migration/V1_28__org_chart_updates.sql]: addressed

**Review by mikebrownccd — APPROVED** [review]
LGTM

**Comment by lukasz-wolski** [comment]
> a COMMIT; at the end would be good
Flyway commits at end of run if successful and most of our migration scripts do not contain a 'COMMIT;' directive

**Pull request merged**